tar/multitape/multitape_delete.c

```diff
@@ -140,7 +149,8 @@ deletetape(TAPE_D * d, uint64_t machinenum, const char * cachedir,
 	close(lockfd);
 
 	/* Success! */
-	return (0);
+	rc = 0;
+	return (rc);
```

Comment:
oh, I thought that if there was any `int rc`, then all exit paths should be `return (rc)` ?  It would certainly simplify things if I can have a `return 0`